parts/linux/cloud-init/artifacts/cse_helpers.sh

```diff
@@ -329,4 +329,14 @@ logs_to_events() {
       return $ret
     fi
 }
+
+should_skip_nvidia_drivers() {
+    body=$(curl -fsSL -H "Metadata: true" --noproxy "*" "http://169.254.169.254/metadata/instance?api-version=2021-02-01")
```

Comment:
kinda gross but not sure if there's a better way to trap the curl exit code while also getting the body parsed out?

Reply:
so this is working off of tags present in the instance metadata?

Reply:
yep, clarified offline